R/produce_source_extract_tests.R

```diff
@@ -28,13 +29,22 @@ produce_source_extract_tests <- function(data,
                                          max_min_vars = c(
                                            "record_keydate1", "record_keydate2",
                                            "cost_total_net", "yearstay"
-                                         )) {
+                                         ),
+                                         hscp_var = TRUE) {
```

Comment:
Could this be called `add_hscp_counts` or something, please? I think `hscp_var` sounds like you should be supplying a variable, not TRUE/FALSE.